### ISSUE TITLE
Missing data

### ISSUE THREAD

**Issue by SlimeDog** [Updated-NoCheatPlus/NoCheatPlus#128]
### Full output of /ncp version
```
>ncp version
[09:50:58] [Server thread/INFO]: »Version information«
[09:50:58] [Server thread/INFO]: Server:
[09:50:58] [Server thread/INFO]: 3188-Spigot-d03d7f1-c851639 ~MC: 1.17.1~
[09:50:58] [Server thread/INFO]: Detected: 1.17.1
[09:50:58] [Server thread/INFO]: NoCheatPlus:
[09:50:58] [Server thread/INFO]: Plugin: 3.17-SNAPSHOT-Updated-b100
[09:50:58] [Server thread/INFO]: MCAccess: 1.13-1.17|? / Bukkit-API
[09:50:58] [Server thread/INFO]: Features:
[09:50:58] [Server thread/INFO]: blocks: BlocksMC1_4, BlocksMC1_5, BlocksMC1_6_1, BlocksMC1_7_2, BlocksMC1_8, BlocksMC1_9, BlocksMC1_10, BlocksMC1_11, BlocksMC1_12, BlocksMC1_13, BlocksMC1_14, BlocksMC1_15, BlocksMC1_16, BlocksMC1_17, MCAccessBukkitModern
[09:50:58] [Server thread/INFO]: checks: FastConsume, Gutenberg, HotFixFallingBlockPortalEnter, AttackFrequency, FlyingFrequency, KeepAliveFrequency
[09:50:58] [Server thread/INFO]: defaults: pvpKnockBackVelocity
[09:50:58] [Server thread/INFO]: packet-listeners: UseEntityAdapter, MovingFlying, OutgoingPosition, KeepAliveAdapter, SoundDistance, WrongTurnAdapter, NoSlow, Fight
[09:50:58] [Server thread/INFO]: Hooks:
[09:50:58] [Server thread/INFO]: AllViolations~NCP~ 1.0, InfiniteJump 1.2.12, ViolationFrequency~NCP~ 1.0
[09:50:58] [Server thread/INFO]: »Related Plugins«
[09:50:58] [Server thread/INFO]: ProtocolLib v4.7.0, ViaVersion v4.0.2-SNAPSHOT
```
### Describe the issue
You're probably aware of this, but just for the record:
```
[08:03:46] [Server thread/INFO]: [NoCheatPlus] Loading NoCheatPlus v3.17-SNAPSHOT-Updated-b100
[08:03:46] [Server thread/INFO]: [NoCheatPlus] onLoad: Early set up of static API, configuration, logging.
[08:03:46] [Server thread/INFO]: [NoCheatPlus] Logging system initialized.
[08:03:46] [Server thread/INFO]: [NoCheatPlus] Detected Minecraft version: 1.17.1
[08:06:54] [Server thread/INFO]: [NoCheatPlus] Enabling NoCheatPlus v3.17-SNAPSHOT-Updated-b100
[08:06:54] [Server thread/INFO]: [NoCheatPlus] McAccess set to: 1.13-1.17|? / Bukkit-API
[08:06:54] [Server thread/INFO]: [NoCheatPlus] Added block-info for Minecraft 1.5 blocks.
[08:06:54] [Server thread/INFO]: [NoCheatPlus] Added block-info for Minecraft 1.6.1 blocks.
[08:06:54] [Server thread/INFO]: [NoCheatPlus] Added block-info for Minecraft 1.7.2 blocks.
[08:06:54] [Server thread/INFO]: [NoCheatPlus] Added block-info for Minecraft 1.8 blocks.
[08:06:54] [Server thread/INFO]: [NoCheatPlus] Added block-info for Minecraft 1.9 blocks.
[08:06:54] [Server thread/INFO]: [NoCheatPlus] Added block-info for Minecraft 1.10 blocks.
[08:06:54] [Server thread/INFO]: [NoCheatPlus] Added block-info for Minecraft 1.11 blocks.
[08:06:54] [Server thread/INFO]: [NoCheatPlus] Added block-info for Minecraft 1.12 blocks.
[08:06:54] [Server thread/INFO]: [NoCheatPlus] Added block-info for Minecraft 1.13 blocks.
[08:06:54] [Server thread/INFO]: [NoCheatPlus] Added block-info for Minecraft 1.14 blocks.
[08:06:54] [Server thread/INFO]: [NoCheatPlus] Added block-info for Minecraft 1.15 blocks.
[08:06:54] [Server thread/INFO]: [NoCheatPlus] Added block-info for Minecraft 1.16 blocks.
[08:06:54] [Server thread/INFO]: [NoCheatPlus] Added block-info for Minecraft 1.17 blocks.
[08:06:54] [Server thread/INFO]: [NoCheatPlus] Applied block patches for multi client protocol support: water_lily, soil, vine, honey_block, shulker_box
[08:06:54] [Server thread/INFO]: [NoCheatPlus] Update block-info: Block shape patch for multi client protocol support around 1.7.x - 1.16.x.
[08:06:54] [Server thread/INFO]: [NoCheatPlus] Inventory checks: FastConsume is available, disabled InstantEat.
[08:06:54] [Server thread/INFO]: [NoCheatPlus] Adding packet level hooks for ProtocolLib (MC 1.17.1)...
[08:06:55] [Server thread/INFO]: [NoCheatPlus] Confirm teleport packet available (via name): TELEPORT_ACCEPT[class=PacketPlayInTeleportAccept, id=0]
[08:06:55] [Server thread/WARN]: [ProtocolLib] [PacketFilterManager] [NoCheatPlus] Unsupported client packet in current Minecraft version: FLYING[PLAY, CLIENT, 254, classNames: [net.minecraft.network.protocol.game.PacketPlayInFlying, net.minecraft.network.protocol.game.ServerboundFlyingPacket, net.minecraft.network.play.client.CPacketPlayer] (unregistered)]
...
[08:07:06] [Server thread/WARN]: [NoCheatPlus] The block data is incomplete:
--- Missing entries -------------------------------
* BLOCK BREAKING (CANDLE_CAKE / SOLID+IGN_PASSABLE+GROUND)
* BLOCK BREAKING (WHITE_CANDLE_CAKE / SOLID+IGN_PASSABLE+GROUND)
* BLOCK BREAKING (ORANGE_CANDLE_CAKE / SOLID+IGN_PASSABLE+GROUND)
* BLOCK BREAKING (MAGENTA_CANDLE_CAKE / SOLID+IGN_PASSABLE+GROUND)
* BLOCK BREAKING (LIGHT_BLUE_CANDLE_CAKE / SOLID+IGN_PASSABLE+GROUND)
* BLOCK BREAKING (YELLOW_CANDLE_CAKE / SOLID+IGN_PASSABLE+GROUND)
* BLOCK BREAKING (LIME_CANDLE_CAKE / SOLID+IGN_PASSABLE+GROUND)
* BLOCK BREAKING (PINK_CANDLE_CAKE / SOLID+IGN_PASSABLE+GROUND)
* BLOCK BREAKING (GRAY_CANDLE_CAKE / SOLID+IGN_PASSABLE+GROUND)
* BLOCK BREAKING (LIGHT_GRAY_CANDLE_CAKE / SOLID+IGN_PASSABLE+GROUND)
* BLOCK BREAKING (CYAN_CANDLE_CAKE / SOLID+IGN_PASSABLE+GROUND)
* BLOCK BREAKING (PURPLE_CANDLE_CAKE / SOLID+IGN_PASSABLE+GROUND)
* BLOCK BREAKING (BLUE_CANDLE_CAKE / SOLID+IGN_PASSABLE+GROUND)
* BLOCK BREAKING (BROWN_CANDLE_CAKE / SOLID+IGN_PASSABLE+GROUND)
* BLOCK BREAKING (GREEN_CANDLE_CAKE / SOLID+IGN_PASSABLE+GROUND)
* BLOCK BREAKING (RED_CANDLE_CAKE / SOLID+IGN_PASSABLE+GROUND)
* BLOCK BREAKING (BLACK_CANDLE_CAKE / SOLID+IGN_PASSABLE+GROUND)
* BLOCK BREAKING (POWDER_SNOW / IGN_PASSABLE)
```

### How to reproduce the issue
- Load up 3.17-SNAPSHOT-Updated-b100
- Read log

### Any possible config options changed or plugins that may cause interference?
None

**Comment by SlimeDog** [comment]
Updated to NoCheatPlus 3.17-b102

Errors remain. On server start-up
```
[08:44:56] [Server thread/INFO]: [NoCheatPlus] Loading NoCheatPlus v3.17-SNAPSHOT-Updated-b102
[08:44:56] [Server thread/INFO]: [NoCheatPlus] onLoad: Early set up of static API, configuration, logging.
[08:44:56] [Server thread/INFO]: [NoCheatPlus] Logging system initialized.
[08:44:56] [Server thread/INFO]: [NoCheatPlus] Detected Minecraft version: 1.17.1
[08:47:03] [Server thread/INFO]: [NoCheatPlus] Enabling NoCheatPlus v3.17-SNAPSHOT-Updated-b102
[08:47:04] [Server thread/INFO]: [NoCheatPlus] McAccess set to: 1.13-1.17|? / Bukkit-API
[08:47:04] [Server thread/INFO]: [NoCheatPlus] Applied block patches for multi client protocol support: water_lily, soil, vine, honey_block, shulker_box
[08:47:04] [Server thread/INFO]: [NoCheatPlus] Update block-info: Block shape patch for multi client protocol support around 1.7.x - 1.16.x.
[08:47:04] [Server thread/INFO]: [NoCheatPlus] Inventory checks: FastConsume is available, disabled InstantEat.
[08:47:04] [Server thread/INFO]: [NoCheatPlus] Adding packet level hooks for ProtocolLib (MC 1.17.1)...
[08:47:04] [Server thread/INFO]: [NoCheatPlus] Confirm teleport packet available (via name): TELEPORT_ACCEPT[class=PacketPlayInTeleportAccept, id=0]
[08:47:04] [Server thread/WARN]: [ProtocolLib] [PacketFilterManager] [NoCheatPlus] Unsupported client packet in current Minecraft version: FLYING[PLAY, CLIENT, 254, classNames: [net.minecraft.network.protocol.game.PacketPlayInFlying, net.minecraft.network.protocol.game.ServerboundFlyingPacket, net.minecraft.network.play.client.CPacketPlayer] (unregistered)]
[08:47:04] [Server thread/INFO]: [NoCheatPlus] Available (and activated) packet level hooks: UseEntityAdapter | MovingFlying | OutgoingPosition | KeepAliveAdapter | SoundDistance | WrongTurnAdapter | Fight
[08:47:04] [Server thread/INFO]: [NoCheatPlus] Force disable FastHeal, FastConsume, PacketFrequency and InstantBow on Minecraft 1.9 and later.
[08:47:04] [Server thread/INFO]: [NoCheatPlus] Version 3.17-SNAPSHOT-Updated-b102 is enabled.
[08:47:04] [Server thread/INFO]: [InfiniteJump] NoCheatPlus hook enabled
[08:47:05] [Craft Scheduler Thread - 24/INFO]: [NoCheatPlus] Could not register an instance for: fr.neatmonster.nocheatplus.components.entity.IEntityAccessLastPositionAndLook
[08:47:05] [Craft Scheduler Thread - 24/INFO]: [NoCheatPlus] Added hook: AllViolations(NCP) [1.0].
[08:47:05] [Craft Scheduler Thread - 24/INFO]: [NoCheatPlus] Added hook: ViolationFrequency(NCP) [1.0].
[08:47:05] [Craft Scheduler Thread - 24/INFO]: [NoCheatPlus] Added hook: InfiniteJump [1.2.12].
[08:47:13] [Server thread/WARN]: [NoCheatPlus] The block data is incomplete:
--- Missing entries -------------------------------
* BLOCK BREAKING (POWDER_SNOW / IGN_PASSABLE)
```

**Comment by xaw3ep** [comment]
We are still not done with PowderSnow yet!

**Comment by SlimeDog** [comment]
Looks to be resolved in NoCheatPlus 3.17-b104. If so, OK to close.
